bears/general/SpaceConsistencyBear.py

```diff
@@ -38,6 +56,11 @@ def run(self,
         result_texts = []
         additional_info_texts = []
 
+        if self.section.aspects and Indentation in self.section.aspects:
+            item = [aspect for aspect in self.section.aspects
+                    if isinstance(aspect, Indentation)][0]
+            use_spaces = item.indent_type == 'space'
```

Comment:
This is an undesired override, ignoring any possibly conflicting explicit `use_spaces` setting, as described in https://coala.zulipchat.com/#narrow/stream/coala/subject/aspects/near/120485662

We should handle such conflicts via `@map_setting_to_aspect`. And we should extend that decorator and the `TasteMeta` class to directly include cases like the above with a syntax like:

```python
@map_setting_to_aspect(
    ...,
    use_spaces=Indentation.indent_type == 'space',
)